README.md

```diff
@@ -11,33 +11,35 @@
     # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
 
-Basic GeoIPS Plugin Template 
+Basic GeoIPS Plugin Template
 =============================
 
-This template repository contains everything necessary to create a fully compatible GeoIPS Plugin Package.
-Each file within this repository contains appropriate modification instructions.
+This template repository contains everything necessary to create a fully
+compatible GeoIPS Plugin Package.  Each file within this repository contains
+appropriate modification instructions.
 
-Follow the 
-[step by step instructions](./docs/template_instructions.rst)
-for modifying the template files within this repo in order to create your own functional plugin.
+Follow the [step by step instructions](./docs/template_instructions.rst) for
+modifying the template files within this repo in order to create your own
+functional plugin.
 
-@ Once this repository has been set up properly, you can remove this "Basic GeoIPS Plugin Template" section in the README.md,
-leaving the appropriate content for your package's README file.
+@ Once this repository has been set up properly, you can remove this "Basic
+GeoIPS Plugin Template" section in the README.md, leaving the appropriate
+content for your package's README file.
 
 
 @package@ GeoIPS Plugin
 ==========================
 
-The @package@ package is a GeoIPS-compatible plugin, intended to be used within the GeoIPS ecosystem.
-Please see the 
-[GeoIPS Documentation](https://github.com/NRLMMD-GEOIPS/geoips/blob/main/README.md)
-for more information on the GeoIPS plugin architecture and base infrastructure.
+The @package@ package is a GeoIPS-compatible plugin, intended to be used within
+the GeoIPS ecosystem.  Please see the
+[GeoIPS Documentation](https://github.com/NRLMMD-GEOIPS/geoips/blob/main/README.md) for
```

Comment:
@jsolbrig @evrose54 @cpcamacho Do you think for now we should just have this point to https://github.com/NRLMMD-GEOIPS/geoips, rather than the full link to /blob/main/README.md ?  We want to make sure whatever link we include here will ALWAYS be valid and useful (since this will get propagated to all manner of plugins).

Reply:
How about this? https://github.com/NRLMMD-GEOIPS/geoips#readme

Reply:
Updated links to point to #readme and #installation!

We just need to commit to keeping geoips README.md updated, and always keeping the "Installation" header.